AppKit/CPCompatibility.j

```diff
@@ -179,7 +182,10 @@ else if (USER_AGENT.indexOf("AppleWebKit/") != -1)
         PLATFORM_BUGS |= CPJavaScriptPasteRequiresEditableTarget;
         // https://bugs.webkit.org/show_bug.cgi?id=39689
         PLATFORM_BUGS |= CPJavaScriptPasteCantRefocus;
+        PLATFORM_FEATURES[CPTextSizingAlwaysNeedsSetFontFeature] = YES;
     }
+	else
+        PLATFORM_ENGINE = CPBlinkBrowserEngine;
```

Comment:
You just need to check the version. The info i got from the internets is that Blink started at Chrome 28.